src/components/player/player.js

```diff
@@ -30,7 +31,10 @@ const createPlayer = (options = {}) => {
     ...{
       muted: preferences.muted ?? true,
       autoplay: preferences.autoplay ?? false,
-      debug: preferences.debug ?? false
+      debug: preferences.debug ?? false,
+      srgOptions: {
+        dataProviderHost: preferences.dataProviderHost || IL_DEFAULT_HOST
```

Comment:
Just to mention that if `dataProviderHost` is undefined it will default to the default IL host by default. But if you want to keep `IL_DEFAULT_HOST` I'm fine with it.

Reply:
Since `IL_DEFAULT_HOST` is the one used in the IL provider of the demo this allows me to have a demo scoped default value that is also propagated to the player.